src/lib/containers/markdown/widget/MarkdownUserOrTeamBadge.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { SynapseConstants } from '../../../utils'
+import { useGetUserGroupHeaderWithAlias } from '../../../utils/hooks/SynapseAPI'
+import TeamBadge from '../../TeamBadge'
+import UserCard from '../../UserCard'
+
+export type MarkdownUserOrTeamBadgeProps = {
+  alias: string
+}
+
+export default function MarkdownUserOrTeamBadge(
+  props: MarkdownUserOrTeamBadgeProps,
+) {
+  const { alias } = props
+  const { data: userGroupheader } = useGetUserGroupHeaderWithAlias([alias])
+
+  return userGroupheader ? (
+    userGroupheader[0].isIndividual ? (
+      <UserCard
+        ownerId={userGroupheader[0].ownerId}
+        size={SynapseConstants.SMALL_USER_CARD}
+      />
+    ) : (
+      <TeamBadge
+        teamId={userGroupheader[0].ownerId}
+        teamName={userGroupheader[0].userName}
+      />
+    )
```

Comment:
Could you use the `UserOrTeamBadge` component here?

Reply:
This was my fault, I led Ellis astray because I forgot we had a UserOrTeamBadge component!